R/OptimInstance.R

```diff
@@ -120,6 +120,41 @@ OptimInstance = R6Class("OptimInstance",
       assert_names(names(y), permutation.of = self$objective$codomain$ids())
       x_domain = transform_xdt_to_xss(xdt, self$search_space)[[1]]
       private$.result = cbind(xdt, x_domain = list(x_domain), t(y)) # t(y) so the name of y stays
+    },
+
+    #' @description
+    #' Evaluates (untransformed) points of only numeric values, and returns a
+    #' scalar objective value, where the return value is negated if the measure
+    #' is maximized. Internally, `$eval_batch()` is called with a single row.
+    #' This function serves as a objective function for optimizers of numeric
+    #' spaces - which should always be minimized.
+    #'
+    #' @param x (`numeric()`)\cr
+    #' Untransformed points.
+    #'
+    #' @return Objective value as `numeric(1)`.
+    objective_function = function(x) {
+      assert_numeric(x, len = self$search_space$length)
```

Comment:
I think this line is redundant becasue the `self$search_space$assert(xs)` already covers that.